### PR TITLE
Some additional Kinect Work

### PR DESCRIPTION
**Pull request by ZappoMan**
- improve the cmake to use the default environment variables as a hint to find the SDK
- make the hand poses include rotation data - automatically expose those to the controller interface
- beginning of (not yet ready) script for wiring up avatar joints to the avatar

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 766c349c655b13062360a4b2a7b449cf5e52b8bd available at: https://gist.github.com/8ba4e39df6992f6c2d59136747ad7a14<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9492-766c349.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9492-766c349.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 35bc6be500fd6fd41b607ad428b6b5edc6bc89a6 available at: https://gist.github.com/8ba4e39df6992f6c2d59136747ad7a14<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9492-35bc6be.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9492-35bc6be.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5400b83aaedaac24e0dcf9961e317ccfafb62d58 available at: https://gist.github.com/8ba4e39df6992f6c2d59136747ad7a14<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9492-5400b83.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9492-5400b83.dmg

**Comment by ZappoMan** [comment]
Test Plan:
- Smoke test -- just double check that nothing broke in hydras, touch controllers, etc... (unlikely)
- Using a Kinect Camera:
    - test that when the plugin is enabled (in the General settings) that the avatars hands (include rotation data) track to the kinect camera

**Comment by hifi-gustavo** [comment]
Build console output for commit: 90b44996aff5ab495f34d26ec5537de8905d7c90 available at: https://gist.github.com/8ba4e39df6992f6c2d59136747ad7a14<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9492-90b4499.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9492-90b4499.dmg

File: plugins/hifiKinect/src/KinectPlugin.cpp [hunk 1]
```diff
@@ -361,38 +381,95 @@ void KinectPlugin::ProcessBody(INT64 time, int bodyCount, IBody** bodies) {
...
+                                //
+                                // To transform from Kinect to our RIGHT Hand.... Negative 90 deg around Y
+                                // 
+                                if (joints[j].JointType == JointType_HandRight) {
+                                    quat kinectToHandRight = glm::angleAxis(-PI / 2.0f, Vectors::UNIT_Y);
+                                    _joints[j].orientation = jointOrientation * kinectToHandRight;
+                                }
+
+                                // To transform from Kinect to our LEFT  Hand.... Postive 90 deg around Y
+                                if (joints[j].JointType == JointType_HandLeft) {
```
Comment (AndrewMeadows) [inline plugins/hifiKinect/src/KinectPlugin.cpp]: Not a blocker but... I'd expect an `else if` statement here.

**Comment by hifi-gustavo** [comment]
Build console output for commit: ea7f66d3cbc2b89a3bd9393dc01304a0f06b386e available at: https://gist.github.com/8ba4e39df6992f6c2d59136747ad7a14<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9492-ea7f66d.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9492-ea7f66d.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: f51de4239632662cd4cab528f7376efcc2bec942 available at: https://gist.github.com/8ba4e39df6992f6c2d59136747ad7a14<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9492-f51de42.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9492-f51de42.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: b14968838c5b6d42c754a2227bb3d0acb0a162ed available at: https://gist.github.com/8ba4e39df6992f6c2d59136747ad7a14<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9492-b149688.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9492-b149688.dmg

**Pull request closed without merging**